lib/ansible/modules/assemble.py

```diff
@@ -48,16 +48,16 @@
     version_added: '1.4'
   remote_src:
     description:
-    - If C(false), it will search for src at originating/master machine.
-    - If C(true), it will go to the remote/target machine for the src.
+    - If V(false), it will search for src at originating/master machine.
+    - If V(true), it will go to the remote/target machine for the src.
     type: bool
     default: yes
     version_added: '1.4'
   regexp:
     description:
-    - Assemble files only if C(regex) matches the filename.
+    - Assemble files only if the given regular expression matches the filename.
     - If not set, all files are assembled.
-    - Every C(\) (backslash) must be escaped as C(\\) to comply to YAML syntax.
+    - Every V(\\) (backslash) must be escaped as V(\\\\) to comply to YAML syntax.
```

Comment:
had doubts about this, but tested and the V(\\) still displays as \ in ansible-doc

Reply:
I thought about this as well; the thing is now that you can escape `\` with `V()` and the other new formatting sequences, seeing `\` inside a `C()` can be confusing - some folks might assume it will escape there as well. So I converted all `C()` with `\` in them to `V()`. (Or at least I hope I did, maybe I overlooked something.)